packages/nextjs/components/Header.tsx

```diff
@@ -25,6 +25,10 @@ export const menuLinks: HeaderMenuLink[] = [
     href: "/debug",
     icon: <BugAntIcon className="h-4 w-4" />,
   },
+  /*{
+    label: "Qulink Profile",
+    href: "/builders/0x926Afc6eB53BBb32B55CEDe50946f2866Dfca2Fc",
+  },*/
```

Comment:
It's bad practice to leave commented code. Other contributors might be confused about whether it should be uncommented or removed.